### PR TITLE
Participation button styles

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: cafebabel/static/css/styles.css [hunk 1]
```diff
@@ -1591,13 +1624,43 @@ textarea#body {
...
+  color: #00283c;
+  position: static;
+}
+
+.article-language-selector li.original-language a:hover {
+  background-color: #b4db13;
+}
+
+.article-language-selector li.original-language a:before {
+  content: 'Original language';
```
Comment (vinyll) [inline cafebabel/static/css/styles.css]: This is inaccessible and will be untranslatable; it should be in the HTML.
As the attribute of the `a` tag maybe? (`<a title="Original language"…>`)

File: cafebabel/static/css/styles.css [hunk 2]
```diff
@@ -1610,6 +1673,9 @@ textarea#body {
     margin-left: 0.5rem;
     vertical-align: middle;
   }
+  .article-language-selector li.original-language a:before {
+    display: block;
```
Comment (vinyll) [inline cafebabel/static/css/styles.css:1677]: I'm not sure why you can't write it in https://github.com/cafebabel/cafebabel.com/pull/272/files#diff-6569206075db87419fc4b34e54f5eb4cR1645.
Just a quick reply will help ;)

File: cafebabel/templates/articles/detail.html [hunk 3]
```diff
@@ -54,7 +54,7 @@ <h1>{{ article.title }}</h1>
       <ul>
         <li><span>{{ article.language }}</span></li>
           {% if article.is_translation %}
-            <li class=original-language>Translated from <a href="{{ article.original_article.detail_url }}">{{ article.original_article.language }}</a></li>
+            <li class=original-language><a href="{{ article.original_article.detail_url }}">{{ article.original_article.language }}</a></li>
```
Comment (vinyll) [inline cafebabel/templates/articles/detail.html]: If you want to play in CSS with this text you could wrap it with a `span`: `<span>Translated from</span> `
(I suppose that's why you moved it from here to CSS but I may have misunderstood)

File: cafebabel/static/css/styles.css (hunk 1 above)
Comment (johangiraud) [reply cafebabel/static/css/styles.css]: Right, done.

File: cafebabel/static/css/styles.css (hunk 2 above)
Comment (johangiraud) [reply cafebabel/static/css/styles.css:1677]: I don't want it to be displayed on mobile neither on click nor "always". And if I use `display:none` always and `display:block` on hover, I can't get any smooth transition. Was that what you meant?

File: cafebabel/templates/articles/detail.html (hunk 3 above)
Comment (johangiraud) [reply cafebabel/templates/articles/detail.html]: I thought it was clearer to have _Original language_ than _Translated from_. What do you think?

File: cafebabel/static/css/styles.css (hunk 2 above)
Comment (vinyll) [reply cafebabel/static/css/styles.css:1677]: Right, makes sense now 👍

File: cafebabel/static/css/styles.css [hunk 4]
```diff
@@ -1643,7 +1643,7 @@ textarea#body {
 }
 
 .article-language-selector li.original-language a:before {
-  content: 'Original language';
+  content: attr(title);
```
Comment (vinyll) [inline cafebabel/static/css/styles.css]: Whaaaatt??? Oh My holly F*** G**! I had no clue we could do that in CSS!
💣

**Review by vinyll — APPROVED** [review]

File: cafebabel/static/css/styles.css (hunk 4 above)
Comment (johangiraud) [reply cafebabel/static/css/styles.css]: haha ! 🚀

**Pull request merged**